readme.md

```diff
@@ -325,7 +325,7 @@ docker-compose exec warp npx ts-node src transpile example_contracts/ERC20.sol
 To test that your contribution doesn't break any features you can test that all previous example contracts transpile and then cairo compile by running the following:
 
 ```bash
-warp test
+yarn test:examples
```

Comment:
can you also move `src/testing.ts` outside of the src folder and move it to `tests`